cogs/randomTopicCog.py

```diff
@@ -0,0 +1,22 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+import random_topic
+
+
+class randomTopicCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    # user types/random_topic and the command will be executed
+    @app_commands.command(name="random-topic")
```

Comment:
Add a description to the command, such as "Send a random conversation topic!" 
![image](https://user-images.githubusercontent.com/101443389/230274348-7bc0abb1-7518-427e-83ac-138daa85a6d3.png)